packages/date-picker/src/DatePicker.scss

```diff
@@ -0,0 +1,131 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/deprecated-component-library-helpers/styles/layout";
+@import "~@kaizen/deprecated-component-library-helpers/styles/color";
+@import "~@kaizen/component-library/styles/mixins/forms";
+@import "~@kaizen/draft-form/KaizenDraft/Form/Primitives/base";
+@import "~@kaizen/draft-button/KaizenDraft/Button/styles";
+
+// -----------------------------------------------
+// Form Input Primitive
+// -----------------------------------------------
+
+// Vars
+$input-height: 48px;
+$input-base-padding-horizontal: $spacing-sm;
+$input-icon-size: 1.25rem; // 20px
+$input-disabled-opacity: 0.3;
+$input-disabled-color-opacity: rgba($color-purple-800-rgb, 0.7);
+
+///////////////////////////////////////////////////
+// BASE STYLES
+///////////////////////////////////////////////////
+
+.wrapper {
+  position: relative;
+}
+
+//  is required to override performance-ui global materialize CSS >:|
+.input {
+  @include form-input-reset;
+  background-color: transparent;
+  background-clip: padding-box;
+  border: $border-solid-border-width $border-solid-border-style transparent;
+  border-radius: $border-solid-border-radius;
+  display: block;
+  font-size: 1rem;
+  line-height: 1.5;
+  padding: 0 $input-base-padding-horizontal;
+  height: $input-height;
+  width: 100%;
+  margin-bottom: 2px;
+
+  &.disabled {
+    &:not(.reversed) {
+      background: $color-white;
+      opacity: $input-disabled-opacity;
+      color: $input-disabled-color-opacity;
+
+      @include form-input-placeholder {
+        color: $input-disabled-color-opacity;
+      }
+    }
+  }
+
+  &:focus:not([disabled]),
+  &:hover:focus:not([disabled]) {
+    @include form-input-placeholder {
+      opacity: 0;
+    }
+  }
+}
+
+.input:focus {
+  outline: $border-focus-ring-border-width $border-focus-ring-border-style
+    $color-blue-500;
+  outline-offset: 2px;
+  pointer-events: none;
+}
+
+///////////////////////////////////////////////////
+// ICON ADORNMENT STYLES
+///////////////////////////////////////////////////
+
+@mixin vertically-center-icon {
+  position: absolute;
+  height: $input-icon-size;
+  top: 50%;
+  transform: translateY(-50%);
+  z-index: 1;
+  margin-top: -2px;
+}
+
+.withStartIconAdornment {
+  .startIconAdornment {
+    @include vertically-center-icon;
+    @include ca-position($start: $ca-grid / 2);
+  }
+
+  .input {
+    @include ca-padding(
+      $start: 1rem + $input-icon-size,
```

Comment:
1rem is to handle the styling for icon at the start of input which isn't relative to `ca-grid`